lib/cn/kvset_split.c

```diff
@@ -550,6 +561,18 @@ vblocks_split(
             if (err)
                 goto errout;
 
+            perfc_inc(pc, PERFC_RA_CNCOMP_RREQS);
+            perfc_inc(pc, PERFC_RA_CNCOMP_WREQS);
+            if (perfc_ison(pc, PERFC_RA_CNCOMP_RBYTES) || perfc_ison(pc, PERFC_RA_CNCOMP_WBYTES)) {
+                struct mblock_props props = { 0 };
+
+                err = mpool_mblock_props_get(ks->ks_mp, src_mbid, &props);
+                ev(err);
+
+                perfc_add(pc, PERFC_RA_CNCOMP_RBYTES, props.mpr_write_len);
+                perfc_add(pc, PERFC_RA_CNCOMP_WBYTES, props.mpr_write_len);
+            }
+
```

Comment:
Use perfc_add2() to increment/add both RREQS/WREQS and RBYTES/WBYTES at the same.
Curious that they're the same?
Why don't you just sum these into some local vars and do a perfc_add2() at the end of the loop?  Would be much less overhead...

Reply:
They're the same because of the clone operation - reading X bytes from the clone source and writing X bytes to the clone target.

Good point regarding summing these into local vars, will update.

Reply:
Cool, then you can do perfc_add2(pc, RREQS, rreqs, RBYTES, rbytes);  and same for WREQS/WBYTES...
It technically requires both counters be active because it only does an ison check on the first counter ID.